dtspec/api.py

```diff
@@ -244,7 +244,12 @@ def _parse_spec(self, json_spec):
 
     def generate_sources(self):
         "Used to generate all source data that will be passed back to the user"
-        for _scenario_name, scenario in self.spec["scenarios"].items():
+        for _scenario_name in self.spec["scenarios"].keys():
```

Comment:
I use underscores to name a variable that is not used (pylint convention, which I haven't added to this repo yet).  So `_scenario_name` wasn't used previously.  Now that you've changed this to loop over just the keys, you should remove the underscore prefix.